meson.build

```diff
@@ -20,6 +20,7 @@ libversion = '@0@.@1@.0'.format(soversion, curversion)
 gaeguli_api_name = '@0@-@1@'.format(meson.project_name(), apiversion)
 
 prefix = get_option('prefix')
+gaeguli_bindir = join_paths(prefix, get_option('bindir'))
```

Comment:
I think better place for auxiliary programs that aren't meant to be run by the users is in `'libexecdir'`.